### PR TITLE
Quick messages

### PR DESCRIPTION
**Pull request by ahendy0**

### CODE REVIEW

**Comment by ahendy0** [comment]
no tests CUZ frick it

File: example/example.py [hunk 1]
```diff
@@ -134,7 +134,20 @@ def webhook():
...
+            )
+            quick2 = QuickReply('location')
+            
+            msg = Message('quick', "My msg text",
+                quick_replies=[
+                    quick,
+                    quick2,
+                ]
+            )
+            print bot.send_message(msg, recipient)
```
Comment (ben-cunningham) [inline example/example.py]: `print`?

Comment (ahendy0) [inline example/example.py]: y-y-y--y-ou too

Comment (ben-cunningham) [inline example/example.py]: Ok

Comment (ahendy0) [inline example/example.py]: sam,e

**Pull request merged**